inst/MetFamily/ui.R

```diff
@@ -2,7 +2,7 @@
 ## constants
 minimumProportionOfLeafs <<- 0.75
 minimumProportionToShowFragment <<- 0.5
-
+crashButton = T
```

Comment:
Just realising that we could read some of those debugging options from environment variables, so the same docker image could be run with or without that enabled.